mrtarget/modules/Evidences.py

```diff
@@ -200,11 +200,21 @@ def validate_evidence(line, process_context):
 
         if efo_id:
             # Check disease term or phenotype term
-            if efo_id not in process_context.kwargs.luts.available_efos:
+            #redis/elasticsearch is based on short ontology id, not full iri
```

Comment:
@afaulconbridge this code about manipulating efo codes should not be here as it has nothing to do with evidence validation but with efo codes/paths/uris. It can be nice if you wrap this with a function and then put it on the efo module or so.